orm.go

```diff
@@ -303,6 +303,11 @@ func (idx *Index) Name() string {
 	return idx.name
 }
 
+// ColumnLabel returns the column label for this index.
+func (idx *Index) ColumnLabel() string {
```

Comment:
column/row labels were deprecated in the server and clients and removed in the cluster-resize compatibility PR at: https://github.com/pilosa/go-pilosa/pull/91 (to be merged when the cluster-resize branch is merged.) Do we need support for them in the enterprise server ?

Reply:
I don't have any issue with dropping them.